schemas/resources/schemas/grid/v0.0.1/metadata.json

```diff
@@ -11,53 +11,64 @@
     "GridDefinition": {
         "CoordinateReferenceSystemID": {
             "Type": "EPSG",
-			      "Code": "EPSG:4326",
+			"Code": "EPSG:4326",
```

Comment:
Spacing is off here.  Code should be under type.

Reply:
fixed.